src/site/xdoc/cmdline.xml.vm

```diff
@@ -169,6 +169,26 @@ java -D&lt;property&gt;=&lt;value&gt;  \
                 <td>
                   <code>-g--generate-xpath-suppression</code>
                 </td>
+                <td>
+                  Generates to output a xpath suppression xml to use to suppress all violations from
+                  user's config. Instead of printing every violation, all violations will be
+                  catched and single suppressions xml file will be printed out. Used only with
```

Comment:
add before "Used only ..."
`Generated suppression will have suppress elements with "checks" and "files" attributes only.`